api/handler/default.go

```diff
@@ -441,6 +399,61 @@ func (m ArcDefaultHandler) processTransaction(ctx context.Context, transaction *
 	}, nil
 }
 
+// processTransactions validates all the transactions in the array and submits to metamorph for processing
+func (m ArcDefaultHandler) processTransactions(ctx context.Context, transactions []*bt.Tx, transactionOptions *api.TransactionOptions) ([]interface{}, error) {
+	span, tracingCtx := opentracing.StartSpanFromContext(ctx, "ArcDefaultHandler:processTransactions")
+	defer span.Finish()
+
+	// validate before submitting array of transactions to metamorph
+	transactionsInput := make([][]byte, 0)
```

Comment:
I think also here memory can be preallocated like this
`transactionsInput := make([][]byte, len(transactions))`